src/handlers/dns/DnsStreamHandler.cpp

```diff
@@ -487,6 +519,7 @@ void DnsMetricsBucket::to_json(json &j) const
     _dns_topUDPPort.to_json(j, [](const uint16_t &val) { return std::to_string(val); });
 
     if (group_enabled(group::DnsMetrics::TopEcs)) {
+        group_enabled(group::DnsMetrics::Counters) ? _counters.queryECS.to_json(j) : void();
```

Comment:
I made this counter metric depend on both `TopEcs` and `Counters` because it needs a deep_sample (processing ECS packets) to be counted..., thoughts @weyrick ?

Reply:
cool!